pod-operation/src/components/wheel_encoder.rs

```diff
@@ -1,76 +1,166 @@
-use std::time::Instant;
+use std::{ops::Sub, time::Instant};
 
 use rppal::gpio::{Gpio, InputPin, Level};
 
 use crate::utils::GpioPins;
 
+const WHEEL_DIAMETER: f32 = 0.25; // feet
+const ENCODER_RESOLUTION: f32 = 16.0; // pulses per revolution
+const DISTANCE_PER_COUNT: f32 = WHEEL_DIAMETER * std::f32::consts::PI / ENCODER_RESOLUTION; // feet
```

Comment:
Are we using feet or meters? If I remember correctly, @ryescholin was using meters for LIDAR measurements.

Reply:
various mechanical ppl want meters